tests/configs/standard/input/config.yaml

```diff
@@ -154,17 +156,26 @@ lab:
       memory: 12Gi
   volumes:
     - containerPath: /home
-      serverPath: /share1/home
-      server: 10.13.105.122
       mode: rw
+      source:
+        type: nfs
+        server: 10.13.105.122
+        serverPath: /share1/home
     - containerPath: /project
-      serverPath: /share1/project
-      server: 10.13.105.122
       mode: ro
+      source:
+        type: hostPath
+        path: /share1/project
     - containerPath: /scratch
-      serverPath: /share1/scratch
-      server: 10.13.105.122
       mode: rw
+      source:
+        type: persistentVolumeClaim
+        storageClassName: sdf-home
+        accessModes:
+          - ReadWriteMany
```

Comment:
We should maybe see if we can consolidate these and the old "ro"/"rw" distinction, but it's not urgent.

Reply:
Hm, or maybe change it so that it exactly matches what Kubernetes does, where there's a boolean `read_only` flag in some places but PVCs get this list of access patterns.